### PR TITLE
DDF-1305: Address all high severity CLM vulnerabilities

### PR DESCRIPTION
**Pull request by ghost**
- Changed version to update hsqldb jar which contained a security vulnerability.

### CODE REVIEW

**Comment by tbatieConnexta** [comment]
@vinamartin @figliold @lessarderic

File: geotools-suite-8.4/pom.xml [hunk 1]
```diff
@@ -78,7 +78,7 @@
 		<dependency>
 			<groupId>org.geotools</groupId>
 			<artifactId>gt-epsg-hsql</artifactId>
-			<version>${bundle.version}</version>
+			<version>13.1</version>
```
Comment (lessarderic) [inline geotools-suite-8.4/pom.xml]: We sure this version (13.1) is compatible with the old one (8.4)? This looks like a huge jump. Couldn't we move everything to 13.1 instead?

Comment (tbatieConnexta) [inline geotools-suite-8.4/pom.xml]: I originally attempted to update all of geotools to 13.1. I spent a lot of time trying to get it to work without success. hsql is only used once in spatial and as far as I can tell is compatible with 8.4. Maybe we should make a ticket for updating geotools to be sure.

**Comment by shaundmorris** [comment]
@tbatieConnexta this won't do what you are expecting. These libraries wrap a released version of geotool-suite and "bundlizes" them. To really update the hsql part would mean we would need to release a new version of this tool suite that the rest of DDF can use.

**Comment by shaundmorris** [comment]
I would suggest we look into updating GeoTools to address this one as a longer term solution. But open to other thoughts on that for sure.

**Comment by shaundmorris** [comment]
@pklinef are you aware any technical reasons to update GeoTools?

**Comment by pklinef** [comment]
We have started to embed newer versions of GeoTools CQL library in some bundles due to timezone bugs.  But that is starting to cause conflicts if you have to also import any other GeoTools package.  We should investigate updating GeoTools to the latest version.

**Comment by shaundmorris** [comment]
@ghost, @pklinef do we want to close this PR in favor of a GeoTools upgrade?

**Comment by shaundmorris** [comment]
The plan is to upgrade GeoTools. https://codice.atlassian.net/browse/DDF-1137

**Pull request closed without merging**